demo-terraform/aws/serverless-app/lambdas.tf

```diff
@@ -34,43 +34,52 @@ resource "aws_iam_role" "lambdas" {
 }
 EOF
 
-    tags = local.project_tags
+  tags = local.project_tags
 
 }
 
 ## Search basic policy for lambdas
 data "aws_iam_policy" "basic_for_lambdas" {
-    name = "AWSLambdaBasicExecutionRole"
+  name = "AWSLambdaBasicExecutionRole"
 }
 resource "aws_iam_role_policy_attachment" "basic_for_lambdas" {
-    for_each = aws_iam_role.lambdas
+  for_each = aws_iam_role.lambdas
 
-    role = each.value.name
-    policy_arn = data.aws_iam_policy.basic_for_lambdas.arn
+  role       = each.value.name
+  policy_arn = data.aws_iam_policy.basic_for_lambdas.arn
 }
 
 resource "aws_lambda_function" "lambda" {
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure VPC access is enabled for AWS Lambda Functions**
 
**Vulnerability**
AWS Lambda functions are not configured to access resources in a Virtual Private Cloud (VPC) which may cause public exposure of private data.
 
**Remediation**
In AWS Console - 
 1. Sign in to AWS Console and go to the Functions page on the Lambda console.
 2. Select a function.
 3. Select Configuration and then choose VPC.
 4. Select Edit.
 5. Select VPC, subnets, and security groups.
 6. Click Save.

source: accurics-